src/app/blogs/[id]/page.tsx

```
@@ -1,24 +1,22 @@
-    <div className="min-h-screen flex flex-col items-center justify-center mt-16">
-      <article className="max-w-2xl space-y-4 font-[family-name:var(--font-geist-sans)]">
-        <h1 className="text-3xl font-bold mb-8 text-[#cecece]">{blog.title}</h1>
-        <Markdown remarkPlugins={[remarkGfm]} children={blog.content} />
-      </article>
-    </div>
-  );
+    return (
+        <div className="min-h-screen flex flex-col items-center justify-center mt-16">
+            <article className="max-w-2xl space-y-4 font-[family-name:var(--font-geist-sans)]">
+                <h1 className="text-3xl font-bold mb-8 text-[#cecece]">{blog.title}</h1>
+                <Markdown remarkPlugins={[remarkGfm]} children={blog.content} />
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)